Makefile

```diff
@@ -658,6 +660,11 @@ test_versionable: install_rs_build_toolchain
 	RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_BUILD_TOOLCHAIN) test --profile $(CARGO_PROFILE) \
 		-p tfhe-versionable
 
+.PHONY: test_backward_compatibility
+test_backward_compatibility: install_rs_build_toolchain
+	TFHE_BACKWARD_COMPAT_DATA_DIR="$(BACKWARD_COMPAT_DATA_DIR)" RUSTFLAGS="$(RUSTFLAGS)" cargo $(CARGO_RS_BUILD_TOOLCHAIN) test --profile $(CARGO_PROFILE) \
+		-p $(TFHE_SPEC) --features=$(TARGET_ARCH_FEATURE),shortint test_backward_compatibility
```

Comment:
keep consistent with other cargo invocation, TFHE_SPEC is at the end of the line

Reply:
fixed